extensions/wikia/ArticleVideo/scripts/jwplayer-autoplay-toggle.js

```diff
@@ -0,0 +1,35 @@
+define('wikia.articleVideo.jwPlayerAutoplayToggle', ['wikia.geo', 'wikia.instantGlobals', 'wikia.cookies'], function (geo, instantGlobals, cookies) {
+
+	return function (playerInstance) {
+		var inAutoplayCountries = geo.isProperGeo(instantGlobals.wgArticleVideoAutoplayCountries),
+			// TODO extract it to separate module
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)